src/apps/service/apps/de/validation.clj

```diff
@@ -51,11 +51,9 @@
   "A tool is publishable if the authenticated user has ownership of the tool and the Docker container that the tool
    uses is in one of the trusted Docker registries."
```

Comment:
Looks like this function's docs need to be updated too?

Reply:
Good catch. Thanks!